prog.go

```diff
@@ -56,6 +56,7 @@ type ProgramSpec struct {
 	AttachTo     string
 	Instructions asm.Instructions
 
```

Comment:
Could the purpose of Flags briefly be documented here?

Reply:
added brief descriptions for both map and prog `Flags`